backend/utils/default_settings.json

```diff
@@ -15,6 +17,10 @@
     "api_key": "",
     "update_interval": 80
   },
+  "jackett": {
+    "url": "",

```

Comment:
thats weird.. this shouldnt be empty on the url.

Reply:
Oops, yeah that needs to be corrected ;)